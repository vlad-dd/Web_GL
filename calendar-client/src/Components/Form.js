import React from "react";

const Form = props => {
  return (
    <div className="form">
      <select onChange={props.selectDay}>
        {props.days.map((day, index) => (
          <option value={index} key={day + index}>{day}</option>
        ))}
      </select>

      <input
        type="text"
        placeholder="What to do"
        name='newEvent'
        value={props.newEvent}
        onChange={props.updateInput}
      />

      <button onClick={props.uploadEvent}>Submit</button>
    </div>
  );
};

export default Form;
