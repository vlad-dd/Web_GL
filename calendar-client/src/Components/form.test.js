import React from 'react';
import Form from './Form'




test('the data is correct ', () => {
    expect('Form').toMatchSnapshot();
});

test('map calls its argument with a non-null argument', () => {
    const mock = jest.fn();
    [1].map(x => mock(x));
    expect(mock).toBeCalledWith(expect.anything());
  });