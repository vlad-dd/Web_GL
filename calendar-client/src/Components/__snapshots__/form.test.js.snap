// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the data is correct  1`] = `"Form"`;

exports[`the login data is correct  1`] = `"Form"`;
